ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.10__okd.yaml

```diff
@@ -59,7 +59,23 @@ tests:
 - as: aws-e2e-operator
   steps:
     cluster_profile: aws
+    env:
+      COMMUNITY: "true"
     test:
+    - as: test
+      commands: |-
+        #!/bin/bash
+        # yq is needed to transform fields in the community bundle
+        curl -L https://github.com/mikefarah/yq/releases/download/v4.13.5/yq_linux_amd64 -o /tmp/yq
+        chmod +x /tmp/yq
+        git clone https://github.com/openshift/windows-machine-config-operator.git
+        cd windows-machine-config-operator
+        make community-bundle
```

Comment:
This seems like its removing running the e2e tests? Won't the commands here override the command present in `windows-e2e-operator-test`?

Reply:
I would think you would need to add the e2e make endpoint in here as well

Reply:
From my understanding of the release repo, wouldn't this be an additional step that runs before the existing workflow (not overriding)?

Reply:
For workflow: manual test overrides test inside workflow. In our case - https://steps.ci.openshift.org/workflow/ipi-aws-ovn-hybrid doesn't have any test configured so the test commands will run and then ref: windows-e2e-operator-test

Reply:
See: [Example of a ci-operator configuration that overrides a workflow field:](https://docs.ci.openshift.org/docs/architecture/step-registry/#ci-operator-test-configuration)